### PR TITLE
Dev/230 ProblemList should be updated when any problem's title, tags is updated or some actions apply to it

### PR DESCRIPTION
**Pull request by Wally5077**
resolve: #230

### CODE REVIEW

File: src/app/problem/ProblemList.js [hunk 1]
```diff
@@ -8,26 +8,46 @@ import {Spinner} from '../commons/Spinner';
...
 const ProblemList = () => {
   const history = useHistory();
   const [problems, setProblems] = useState(undefined);
   const [showCreateProblemModal, setShowCreateProblemModal] = useState(false);
-  const [problemTabActiveIndex, setProblemTabActiveIndex] = useState(1);
-  const fetchAllProblems = useCallback(() => {
-    problemService.getAllProblems()
-      .then(setProblems);
-  }, [setProblems]);
+  const [problemTabActive, setProblemTabActive] = useState('Problems');
```
Comment (Chaoyuuu) [inline src/app/problem/ProblemList.js:21]: rename to `setActiveProblemTab`

**Review by Chaoyuuu — CHANGES_REQUESTED** [review]
Review done!

**Review by Chaoyuuu — CHANGES_REQUESTED** [review]
LGTM

**Review by Chaoyuuu — APPROVED** [review]

**Review by timt1028 — APPROVED** [review]
LGTM

**Pull request merged**